collectd_mesos_plugin.py

```diff
@@ -215,8 +215,10 @@ def run(self):
                 task = value.replace(".", "_")
             if name == 'SHUB_JOBKEY':
                 kumo_job = value
+            if name == 'MARATHON_APP_DOCKER_IMAGE':
+                docker_image = value
 
-        if kumo_job:
+        if kumo_job and "kumo" in docker_image:
```

Comment:
@cyberplant: what motivated this change?

Kumo jobs doesn't have MARATHON_APP_DOCKER_IMAGE set, they are not managed by Marathon so docker_image will be always `None` and raises an error in this line.